resources/qml/MainWindow/MainWindowHeader.qml

```diff
@@ -55,9 +57,10 @@ Rectangle
                 text: model.name.toUpperCase()
                 checkable: true
                 checked: model.active
+                anchors.verticalCenter: parent.verticalCenter
                 exclusiveGroup: mainWindowHeaderMenuGroup
                 style: UM.Theme.styles.main_window_header_tab
-                height: UM.Theme.getSize("main_window_header").height
+                height: Math.round(0.56 * UM.Theme.getSize("main_window_header").height)
```

Comment:
Hmm, bit of a magic number. Where is this coming from?

Reply:
This is because the header height by default is 50px and the buttons has to be 28px following the designs. If for some reason the size of the header changes, then the buttons will keep the proportion. Maybe is better to have a margin on top and bottom, or they will always have a fixed height????
Ping @AnneUltimaker, @LukeChen-Ultimaker and Yi-Ann.

Reply:
I think it's best to use the same size as the rest of our buttons (an existing theme item), even if that deviates a bit from the original design. Ultimately it will make the application easier to maintain, and re-using the same sizes helps to make the application feel more consistent.

Second best would be to use a separate theme item to give it a fixed but different height. If we ever resize the header bar, we'll probably also want to give this a different ratio anyway so that the button doesn't get deformed.

Reply:
Done in b6ea0e638578daa32fcf383335b112e8efcce7e8.

The height of the button has to be 28px as indicated by our designers.

Reply:
On my external screen it's currently 32 pixels high. On my laptop's built-in screen it's currently 48 pixels high.

You are NEVER going to get this pixel perfect.